prestoadmin/collect.py

```diff
@@ -45,7 +45,7 @@
 
 TMP_PRESTO_DEBUG = '/tmp/presto-debug/'
 TMP_PRESTO_DEBUG_REMOTE = '/tmp/presto-debug-remote'
-OUTPUT_FILENAME_FOR_LOGS = '/tmp/presto-debug-logs.tar.gz'
+OUTPUT_FILENAME_FOR_LOGS = 'presto-debug-logs.tar.gz'
 OUTPUT_FILENAME_FOR_SYS_INFO = '/tmp/presto-debug-sysinfo.tar.gz'
```

Comment:
it would be nice to update this too

Reply:
@kokosing, my code doesn't use those variables - I think it will broke some tests...
I can apply the same logic for all the collect commands and then change it.